types/npmcli__package-json/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for @npmcli/package-json 2.0
+// Project: https://github.com/npm/package-json
+// Definitions by: Michaël De Boey <https://github.com/MichaelDeBoey>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import type { PackageJson } from './package-json';
+
+export type Content = PackageJson;
+
+declare class NPMCliPackageJson {
+    constructor(path: string);
+
+    static load: (path?: string) => Promise<NPMCliPackageJson>;
```

Comment:
```suggestion
    /** @async */
    static load: (path?: string) => Promise<NPMCliPackageJson>;
```
as the `.d.ts` cannot contain `async` keyword:
https://jsdoc.app/tags-async.html

Reply:
and there should be an instance method returning same results

Reply:
Done!